test_example_problem.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from example_problem import tip_calculator
+
+class TipTestCase(unittest.TestCase):
+    """Test for calculating tips at restaurants"""
+
+    def test_tip_calculator(self):
+        """Test to see if calculator returns the correct value"""
+
+        bill = 100
```

Comment:
Awesome job here!!! One thing I'd recommend getting in the habit of is separating the phases of the tests with blank lines:

```
# setup
bill = 100
tip_rate = 20

# exercise code
total_tip = tip_calculator(bill, tip_rate)

# verify the results
self.assertEqual(total_tip, 20)
```

See this video for a good run-down: https://thoughtbot.com/upcase/videos/four-phases-of-testing